src/io/flutter/preview/PreviewView.java

```diff
@@ -205,6 +222,15 @@ public void initToolWindow(@NotNull ToolWindow toolWindow) {
     tree.setCellRenderer(new OutlineTreeCellRenderer());
     tree.expandAll();
 
+    tree.addMouseListener(new PopupHandler() {
+      public void invokePopup(Component comp, int x, int y) {
+        final ActionManager actionManager = ActionManager.getInstance();
+        final ActionGroup group = createTreePopupActions();
+        final ActionPopupMenu popupMenu = actionManager.createActionPopupMenu(ActionPlaces.UNKNOWN, group);
+        popupMenu.getComponent().show(comp, x, y);
```

Comment:
If the group is empty, does anything show? Should we just not call toe show() method in those cases?

Reply:
Yes, we did show.
I'm fixing this.